### PR TITLE
K8SPXC-457: Wait for certmanager certificate issue

### PR DESCRIPTION
**Pull request by qJkee**
[K8SPXC-457](https://jira.percona.com/browse/K8SPXC-457)

Test are falling because they can't find required secrets

### CODE REVIEW

File: e2e-tests/monitoring-2-0/conf/monitoring.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   secretsName: my-cluster-secrets
   sslSecretName: some-name-ssl
+  sslInternalSecretName: some-name-ssl-internal
```
Comment (delgod) [inline e2e-tests/monitoring-2-0/conf/monitoring.yml:8]: the internal secret is optional. please don't add it

File: e2e-tests/monitoring/conf/monitoring.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   secretsName: my-cluster-secrets
   sslSecretName: some-name-ssl
+  sslInternalSecretName: some-name-ssl-internal
```
Comment (delgod) [inline e2e-tests/monitoring/conf/monitoring.yml:8]: the internal secret is optional. please don't add it

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| big-data | passed |
| recreate | passed |
| haproxy | passed |
| scaling | passed |
| init-deploy | passed |
| scaling-proxysql | failed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| affinity | failed |
| self-healing | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| one-pod | passed |
| demand-backup-encrypted-with-tls | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-591-5050fcd1

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| recreate | passed |
| storage | passed |
| haproxy | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |

**Review by hors — APPROVED** [review]

**Review by blez — APPROVED** [review]

**Pull request merged**